cc-backend/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import express, { Express, Request, Response, NextFunction } from "express";
+import swaggerUi from "swagger-ui-express";
+import YAML from "yamljs";
+import dotenv from "dotenv";
+import axios from "axios";
+import * as OpenApiValidator from "express-openapi-validator";
+import sysRouter from "./routes/systemRoutes";
+import itemRouter from "./routes/itemRoutes";
+import advertRouter from "./routes/advertRoutes";
+import userRouter from "./routes/userRoutes";
+
+import { Error } from "./types";
+import { Item } from "./types";
+import { UserInfo } from "./types";
+import { Config } from "./types";
```

Comment:
I think these can be written as:
import {
  Error,
  Item,
  UserInfo,
  Config} from "./types";

Reply:
That is true and I've used that in other places. I will fix these later as I hopefully have more time next week. Some imports are also not needed so gotta check them too.